pop/service/service_test.go

```diff
@@ -24,22 +24,38 @@ func TestMain(m *testing.M) {
 	log.MainTest(m)
 }
 
+func svc(t *testing.T, local *onet.LocalTest, n int) *Service {
+	servers := local.GenServers(n)
+	services := local.GetServices(servers, serviceID)
+	if services[0] == nil {
+		t.Log("The service is not available to test due to incompatible suite.")
+		return nil
```

Comment:
are we able to do something like `t.Skip` when this happens, I think it'll be cleaner than having `service == nil` in every test.

Reply:
That's even a
```
require.NotNil(t, services[0], "incompatible suite")
```
no?

Reply:
It is not a fatal error: it is just that the current definition of cothority.Suite does not allow this service to be instantiated, so testing it is not possible. t.Skip is the right thing.